data_owner_ids.py

```diff
@@ -101,7 +102,7 @@ def do_data_owner_ids(c):
         print("More than one link_id file found")
         print(link_ids)
         link_id_times = [
-            datetime.strptime(x.name[-19:-4], "%Y%m%dT%H%M%S") for x in link_ids
+            datetime.strptime(x.name[-19:-4], TIMESTAMP_FMT) for x in link_ids
```

Comment:
Looks like we would need to be aware of this line if we do change TIMESTAMP_FMT -- the length of the snippet being parsed out will depend on the date format. Can you think of a clean way we can handle that here? Unfortunately the length of a timestamp will be different than the length of the format string (for example `len("2022-11-10") !=  len("%Y-%m-%d")` )